### DISCUSSION TITLE
Mouse cursor flickering/disappearing

### DISCUSSION THREAD

**Discussion by Ioncloud9** [voxeltycoon/issues#596] [UI&UX]
**Describe the bug**
I cant really say when the issue started, just all of a sudden my mouse cursor started flickering to the point I was unable to see it. I backed out to the main menu and started playing with some of the options, turning on vsync seems to fix it on the main menu; however, when I load back into my game, the cursor is flickering/missing. I can still click on things if I'm trying hard. I tried toggling vsync while in-game and that did not seem to help.

**Steps to reproduce**
Not sure what caused it in the first place. I didn't change any settings between it being fine and it starting, the issue does persist between reloads.

**Build version**
v0.85

**Platform**
Windows 10

**Reproducibility**
Once it started, it continues to be an issue in game across restarts

**Comment by Ioncloud9** [comment]
ah, if I pause the game the mouse cursor stops flickering and is fine... if I un-pause it starts flickering/goes missing...

**Comment by Blanus-1** [comment]
I wanted to report the same issue! I also can't really say what started the error!

**Comment by BadRobotGT** [comment]
i have the same problem too, sadly i can't help either with the issue...

**Comment by Ioncloud9** [comment]
I figured it out... I upped the UI scale to 200%, when I put it back down to 150% the issue clears up... (playing at 4k, some of the stuff is still small...)

**Comment by Blanus-1** [comment]
I also play on 200% 4k. Did you try to scale to 199% doesd this also fix the error. I reinstalled now and this also fixes it

**Comment by Ioncloud9** [comment]
nm, it issue came back after putting it back to 150% and playing for a bit

**Comment by BadRobotGT** [comment]
I did an NVIDIA-Driver update, now it seems to be solved, i hope this helps you also...

**Comment by dsonck92** [comment]
I also have this issue, however:

- Updating the nvidia driver will reduce the issue a lot, but it's not gone. (I have a particular area in the game where I can consistently reproduce it)
- Changing the Windows 10 scaling option from 200 to 100% makes the issue basically go away for me. I want to test this out on TPF2 as well, because that game has a cursor setting that makes the cursor flicker very similar to Voxel Tycoon.

**Comment by austinbarkley** [comment]
+1 occurring at 4k 200% UI scale
